pkg/project/project.go

```diff
@@ -5,7 +5,11 @@ var (
 	gitSHA      = "n/a"
 	name        = "awscnfm"
 	source      = "https://github.com/giantswarm/awscnfm"
-	version     = "12.1.1"
+	// version is synchronnized with the latest Giant Swarm release version so
+	// that we know which version we use as base for conformance testing. Please
+	// make sure that this version changes with each new release so that we stay
+	// synchronized with the latest AWS release we publish.
```

Comment:
It should be mentioned that both the major and minor version have to match (not the patch version).

Reply:
How does the release cycle here work then? Why do we have a x.x.1 patch version here? Will `awscnfm` always be behind or ahead of the actual Giant Swarm release?

Reply:
Don't know exactly. But we need a way to release a fix for mistakes in awscnfm, independent of the tenant cluster release cycle. My thinking is also that the desired behaviour of a tenant cluster should not change from patch version to patch version.